platform/darwin/src/MGLRendererConfiguration.mm

```diff
@@ -0,0 +1,119 @@
+        // Also support NSString to correspond with the behavior of `-[NSUserDefaults boolForKey:]`
+        return [infoDictionaryObject boolValue];
+    }
+    return NO;
+}
+
+- (MGLGlyphsRasterizationMode)glyphsRasterizationMode {
+    id infoDictionaryObject = [NSBundle.mainBundle objectForInfoDictionaryKey:MGLGlyphsRasterizationModeKey];
+    return [self glyphsRasterizationModeWithInfoDictionaryObject:infoDictionaryObject];
+}
+
+- (MGLGlyphsRasterizationMode)glyphsRasterizationModeWithInfoDictionaryObject:(id)infoDictionaryObject {
+    if (!infoDictionaryObject || ![infoDictionaryObject isKindOfClass:[NSString class]]) {
+        return MGLGlyphsRasterizationModeNone;
+    }
+    NSDictionary *nameOptionMap = @{@"MGLNoGlyphsRasterizedLocally":@(MGLGlyphsRasterizationModeIdeographsRasterizedLocally),
+                                    @"MGLIdeographsRasterizedLocally":@(MGLGlyphsRasterizationModeNoGlyphsRasterizedLocally),
+                                    @"MGLAllGlyphsRasterizedLocally":@(MGLGlyphsRasterizationModeAllGlyphsRasterizedLocally)};
+
+    return (MGLGlyphsRasterizationMode)[nameOptionMap[infoDictionaryObject] integerValue];
+}
+
+- (mbgl::GlyphsRasterizationOptions)glyphsRasterizationOptions {
+    return [self glyphsRasterizationOptionsWithLocalFontFamilyName:self.localFontFamilyName rasterizationMode:self.glyphsRasterizationMode];
+}
+
+- (mbgl::GlyphsRasterizationOptions)glyphsRasterizationOptionsWithLocalFontFamilyName:(nullable NSString *)fontFamilyName
+                                                                    rasterizationMode:(MGLGlyphsRasterizationMode)rasterizationMode {
+    mbgl::GlyphsRasterizationOptions options;
+    if (fontFamilyName == nil) {
+        if (rasterizationMode != MGLGlyphsRasterizationModeNoGlyphsRasterizedLocally) {
+            MGLLogError(@"The `MGLIdeographicFontFamilyName` is set to `NO`, this will make `MGLGlyphsRasterizationMode` always be `MGLNoGlyphsRasterizedLocally`.");
+        }
+        options.rasterizationMode = mbgl::GlyphsRasterizationMode::NoGlyphsRasterizedLocally;
+        return options;
+    }
+    
+    options.fontFamily =  mbgl::optional<std::string>(std::string(fontFamilyName.UTF8String));
+    switch (rasterizationMode) {
+        case MGLGlyphsRasterizationModeIdeographsRasterizedLocally:
+            options.rasterizationMode = mbgl::GlyphsRasterizationMode::IdeographsRasterizedLocally;
+            break;
+        case MGLGlyphsRasterizationModeNoGlyphsRasterizedLocally:
+            options.rasterizationMode = mbgl::GlyphsRasterizationMode::NoGlyphsRasterizedLocally;
+            break;
+        case MGLGlyphsRasterizationModeAllGlyphsRasterizedLocally:
+            options.rasterizationMode = mbgl::GlyphsRasterizationMode::AllGlyphsRasterizedLocally;
+            break;
+        default:
+            options.rasterizationMode = mbgl::GlyphsRasterizationMode::NoGlyphsRasterizedLocally;
```

Comment:
How about mode `MGLGlyphsRasterizationModeNone`? 
And should we print an error log in case that the mode fallback to default?

Reply:
Yes, we should always fallback to default. If `fontFamilyName` is not nil and user sets [`MGLNoGlyphsRasterizedLocally`](https://github.com/mapbox/mapbox-gl-native-ios/pull/517/files#diff-5088f83de2b6336d6e2b089e057d5dc2a6c4025d5809fe29e50d0cd307dd4992R104) or `MGLGlyphsRasterizationModeNone`, I think we should warn and fallback to `MGLIdeographsRasterizedLocally`

Reply:
We may want to revisit and test the behavior here once beta is released.